packages/astro/src/@types/astro.ts

```diff
@@ -733,6 +733,27 @@ export interface AstroUserConfig {
 		 * ```
 		 */
 		serverEntry?: string;
+		/**
+		 * @docs
+		 * @name build.inlineStylesheets
+		 * @type {('always' | 'auto' | 'never')}
+		 * @default `never`
+		 * @version 2.4.0
```

Comment:
```suggestion
		 * @version 2.6.0
```
I think this should be 2.6 because any mention of it being an experimental option is going away. So someone with e.g. 2.4 or 2.5 but who never enabled the flag would not have access to this.

Reply:
Yeah, this makes sense. Thanks!